exporter/osg/osgdata.py

```diff
@@ -1597,9 +1623,10 @@ def needBake(self, blender_object):
 
     def handleMorphAnimationBaking(self, is_multi_animation=False):
         Log("Exporting morph animation on object {}".format(self.object.name))
-        if not self.has_morph:
+        if not self.has_morph or self.object.data.shape_keys.use_relative and not self.config.bake_animations:
```

Comment:
the condition on shape keys relativity seems strange to me; is it really what you want? why?